jetbrains-core/tst/software/aws/toolkits/jetbrains/ui/tree/AsyncTreeModelTest.java

```diff
@@ -55,6 +58,9 @@
    */
   private static final boolean PRINT = false;
 
+  @Rule
+  public ProjectRule projectRule = new ProjectRule();
```

Comment:
would it make sense for our tests to share a base class that performs this kind of common setup?

Reply:
It is expensive, so not all tests want it